### ISSUE TITLE
Question : Is LXC Bench supported for android

### ISSUE THREAD

**Issue by 1kapilgupta1** [gmacario/lxcbench#1]
please help me to know if LXC bench is supported for Android also?
If yes, is that already available on GitHub?

**Comment by gmacario** [comment]
Hello @1kapilgupta1,

Thanks for your interested in LXCbench.

> Please help me to know if LXC bench is supported for Android also?

Unfortunately I have changed my job and Android is currently not a priority, so my original plans to port LXCbench to Android never materialized.

There are probably other options you may have a look at, but first of all let me understand: What is your exact use case you are interested in?

**Comment by 1kapilgupta1** [comment]
I want to benchmark native android and android with container.

I heard that Phoronix is also not supported for android, Do you have any idea about any other options.

**Comment by gmacario** [comment]
> I want to benchmark native android and android with container.

Let me rephrase to make sure I understood it correctly: You want to "benchmark performances of Android application running on a native Android stacl vs. the same Android stack running inside a Linux container.

If so you should probably just install an Android stack on the hardware platform you are interested in and run some Android applications which provide you the performance indications you are interested in, then compare the results on the same application running on the same Android stack but executed inside a Linux container.

> I heard that Phoronix is also not supported for android, Do you have any idea about any other options.

I have never tried Phoronix on Android but based on what I understood the tests are natively compiled on the target platform, therefore I tend to agree that it will be difficult to compile them natively on an Android system (i.e. lack of gcc, glibc, etc.)

Hope that (somewhat) helps,
Gianpaolo
